src/Component/login.js

```diff
@@ -0,0 +1,124 @@
+import { onNavigate } from '../main.js';
+import { iniciarSesion, usuarioExistente, google } from '../lib/index.js';
+import { modal } from '../Component/modal.js';
+
+export const login = () => {
+  const seccionGeneral = document.createElement('section');
+  const tabla = document.createElement('IMG');
+  tabla.src = './images/login.png';
+  tabla.setAttribute('id', 'cocina');
+  tabla.setAttribute('alt', 'Tabla de cocina');
+  tabla.setAttribute('class', 'desktop');
+
+  const seccionLogin = document.createElement('section');
+  seccionLogin.setAttribute('id', 'login');
+
+  const logoYummi = document.createElement('IMG');
+  logoYummi.src = './images/logo.png';
+  logoYummi.setAttribute('id', 'logoinicio');
+  logoYummi.setAttribute('alt', 'Yummy, bienvenido a tu mundo de cocina');
+
+  const inputEmailLogin = document.createElement('input');
+  inputEmailLogin.setAttribute('type', 'email');
+  inputEmailLogin.setAttribute('id', 'email');
+  inputEmailLogin.setAttribute('placeholder', 'Email');
+
+  const inputContraseñaLogin = document.createElement('input');
+  inputContraseñaLogin.setAttribute('type', 'password');
+  inputContraseñaLogin.setAttribute('id', 'contraseña');
+  inputContraseñaLogin.setAttribute('placeholder', 'Contraseña');
+  const btnmostrarcontraseña = document.createElement('button');
+  btnmostrarcontraseña.setAttribute('id', 'ojito');
+  const ojo = document.createElement('IMG');
+  ojo.src = './images/ojitobueno.png';
+  btnmostrarcontraseña.appendChild(ojo);
+  // btnmostrarcontraseña.textContent = 'mostrar Contraseña';
+  btnmostrarcontraseña.addEventListener('click', () => {
+    if (inputContraseñaLogin.type === 'password') {
+      inputContraseñaLogin.type = 'text';
+    } else {
+      inputContraseñaLogin.type = 'password';
+    }
+  });
+
+  const botonLogin = document.createElement('button');
+  botonLogin.setAttribute('id', 'loginbo');
+  botonLogin.setAttribute('class', 'botones');
+  const linkbotonlogin = document.createElement('a');
+  linkbotonlogin.setAttribute('href', '#contmodal');
+  linkbotonlogin.textContent = 'Iniciar sesión';
+  botonLogin.appendChild(linkbotonlogin);
+  botonLogin.addEventListener('click', () => {
+    usuarioExistente();
+    iniciarSesion();
+  });
+  const imgGoogle = document.createElement('IMG');
+  imgGoogle.src = './images/google.png';
+  imgGoogle.setAttribute('alt', 'google');
+  imgGoogle.setAttribute('id', 'iconogoo');
+
+  const botonGoogle = document.createElement('button');
+  botonGoogle.setAttribute('id', 'google');
+  botonGoogle.setAttribute('class', 'botones');
+  botonGoogle.textContent = 'Iniciar con Google';
+  botonGoogle.appendChild(imgGoogle);
+  botonGoogle.addEventListener('click', () => {
+    google();
+  });
+
+  const botonContraseña = document.createElement('button');
+  botonContraseña.setAttribute('class', 'invisible');
+  botonContraseña.textContent = '¿Olvidaste tu contraseña?';
+  botonContraseña.setAttribute('id', 'letras1');
```

Comment:
esta raro el id no ?
```suggestion
  botonContraseña.setAttribute('id', 'letters');
```